openff/evaluator/server/server.py

```diff
@@ -434,6 +457,8 @@ def _prepare_batches(self, submission, request_id):
             batches = self._batch_by_same_component(submission, force_field_id)
         elif batch_mode == BatchMode.SharedComponents:
             batches = self._batch_by_shared_component(submission, force_field_id)
+        elif batch_mode == BatchMode.NoBatch:
+            batches = self._no_batch(submission, force_field_id)
```

Comment:
Could you add a test that covers these lines?

Reply:
Sure -- added a test for _prepare_batches.

Reply:
PS This code path should be used in the tests in #604 and #608.